docs/compute/modules.md

```diff
@@ -88,7 +88,7 @@ Command                 | Shortened Command            | Description  | Example
 ## Loading Modules in a Job Script
```

Comment:
At the beginning of this there is a video that is outdated. I think it is fine to leave it in for now, but we should create a GitHub issue for its replacement/removal